zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -783,6 +783,15 @@ public void setInfo(Map<String, Object> info) {
     this.info = info;
   }
 
+  public void setSequentialRunStatus(boolean runStatus) {
+    Map<String, Object> infoMap = getInfo();
+    infoMap.put("isRunningSequentially", runStatus);
```

Comment:
Could we change it to more general word `isRunning`? Because what we want to achieve is disallow any write operation on note when it is running. And we may introduce other kinds of running mode in future.

Reply:
No objections

Reply:
@zjffdu, done.